src/components/LoadingMessage/LoadingMessage.less

```diff
@@ -0,0 +1,33 @@
+.lucid-LoadingMessage {
+
+	.lucid-Icon {
```

Comment:
why is this rule-set above all the declarations while the other nested rule-sets are below the declarations?